### ISSUE TITLE
A few questions about Yaru on Ubuntu Bionic

### ISSUE THREAD

**Issue by vaskark** [ubuntu/yaru#653]
Sorry if this is obvious or frivolous. I've installed Yaru by cloning the git repo and doing the meson build, ninja, sudo ninja install routine and everything seems fine. Yet there is no option to set Yaru as my gnome-shell theme. I do, however, have a Yaru Ubuntu session at the gdm screen, but selecting that doesn't seem to do anything. I also had this problem when I tried using the communitheme snap package. Can anyone point out what I'm doing wrong, if anything? Also, is it possible to use a dark variant of Yaru's gtk themes? And if so, how can I go about activating it?

Thanks in advance. And if this isn't the place for this type of question ... please delete :)

**Comment by Feichtmeier** [comment]
Which --prefix did you use?
I had similar problems when using the wrong prefix
What works is --prefix=/usr

**Comment by vaskark** [comment]
Well, I didn't change the prefix. /usr was the default as stated in the meson.build file so that's what was used. The gtk, cursor, and icon parts all work fine. But usually the gnome-shell stuff gets put in /usr/share/themes/"theme_folder" location, but it isn't listed there -- instead it's in /usr/share/gnome-shell. It does with other themes, anyway, and they all work fine for me.

I also tried using the ppa. Same problems.

**Comment by drakkan** [comment]
the patch here:

https://aur.archlinux.org/cgit/aur.git/tree/0001-gnome-shell-fix-install-dir.patch?h=yaru

allows to set Yaru gnome shell theme

**Comment by clobrano** [comment]
> the gnome shell theme is installed in "/usr/share/themes/Yaru/gnome-shell" that should be the right location

@drakkan could you point me to the documentation about this?

I am not sure if it's something distro dependent, but on Ubuntu under /usr/share/themes/Adwaita there is no gnome-shell as well, so I guess the meson installation path for Ubuntu is right

**Comment by clobrano** [comment]
@vaskark could you paste the build log?

**Comment by drakkan** [comment]
@clobrano please take a look at fedora gnome shell theme such as this one

http://mirror2.hs-esslingen.de/fedora/linux/releases/28/Everything/x86_64/os/Packages/g/gnome-shell-theme-selene-3.4.0-14.fc28.noarch.rpm

and please take a look to tweak tool source code:

https://gitlab.gnome.org/GNOME/gnome-tweaks/blob/master/gtweak/tweaks/tweak_group_appearance.py#L225

**Comment by clobrano** [comment]
> on ubuntu are you able to set yaru gnome shell theme using gnome-tweak-tools?

of course, this is what we're doing since the beginning of the project, but I wonder if we are mixing problems.
The bug name report the problem on Ubuntu Bionic, your patch refers to Arch and seems valid for Fedora as  well, is that correct?
@vaskark sorry for the silly question, but you're running ubuntu when you've got the problem right?

**Comment by vaskark** [comment]
@clobrano Yeah, I'm on ubuntu bionic. Sorry for not getting back sooner.
No, I cannot set the yaru gnome-shell theme in gnome-tweaks. This is the entire problem, not only with building from git but also the ppa and snap install methods. It's just never worked for me. Also, I do not know how to use the patch that someone kindly posted above. I'm also baffled as to why there needs to be a theme choice at the login screen. No other theme works like that.

**Comment by vaskark** [comment]
Also, I found a way to set the dark theme:

cd /usr/share/themes/Yaru/gtk-3.0
sudo mv gtk.css gtk.css.backup
sudo ln -s gtk-dark.css gtk.css

Looks pretty good ;)
